.github/workflows/pre-merge-checks.yml

```diff
@@ -202,14 +202,17 @@ jobs:
       - run-integration-tests
```

Comment:
Is there a reason we haven't got the if statement just for the job?

Reply:
It's a required check, so need it to pass

Reply:
Ah sure, sounds good